src/workers/cdc.ts

```diff
@@ -225,6 +227,25 @@ const onAlertsChange = async (
   }
 };
 
+const onNotificationsChange = async (
+  con: DataSource,
+  logger: FastifyLoggerInstance,
+  data: ChangeMessage<Notification>,
+): Promise<void> => {
+  if (['c'].includes(data.payload.op)) {
```

Comment:
I find this version more readable

```suggestion
  if (data.payload.op === 'c') {
```

Reply:
Makes sense for the singular one, will refactor